sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTest.java

```diff
@@ -2379,6 +2379,37 @@ public void analyzePiiEntityRecognitionWithCategoriesFilters(HttpClient httpClie
         );
     }
 
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.textanalytics.TestUtils#getTestParameters")
+    public void analyzePiiEntityRecognitionWithDomainFilters(HttpClient httpClient,
+        TextAnalyticsServiceVersion serviceVersion) {
+        client = getTextAnalyticsClient(httpClient, serviceVersion);
+        analyzePiiEntityRecognitionWithDomainFiltersRunner(
+            (documents, tasks) -> {
+                SyncPoller<AnalyzeActionsOperationDetail, PagedIterable<AnalyzeActionsResult>> syncPoller =
+                    client.beginAnalyzeActions(documents, tasks,
+                        new AnalyzeActionsOptions().setIncludeStatistics(false), Context.NONE);
+                syncPoller = setPollInterval(syncPoller);
+                syncPoller.waitForCompletion();
+                PagedIterable<AnalyzeActionsResult> result = syncPoller.getFinalResult();
+
+                validateAnalyzeBatchActionsResultList(false, false,
+                    Arrays.asList(getExpectedAnalyzeBatchActionsResult(
+                        IterableStream.of(Collections.emptyList()),
+                        // TODO: Organization "Microsoft" should be displayed since it is not a SSN.
```

Comment:
nit: shouldn't be display

Reply:
actually, I don't know what u mean... you are trying to do PHI here, but you are talking about SSN so idk

Reply:
I might be totally wrong on PHI filter. I thought `Microsoft employee` is not in PHI categroy.

Reply:
In the https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/named-entity-types?tabs=personal#category-organization

`Microsoft` is the type of Organization, it is PHI, but `employee` is the type of PersonType, it is not PHI.